extensions/ql-vscode/src/pure/messages.ts

```diff
@@ -972,7 +1003,10 @@ export const checkUpgrade = new rpc.RequestType<WithProgressId<UpgradeParams>, C
  * Compile an upgrade script to upgrade a dataset.
  */
 export const compileUpgrade = new rpc.RequestType<WithProgressId<CompileUpgradeParams>, CompileUpgradeResult, void, void>('compilation/compileUpgrade');
-
+/**
+ * Compile an upgrade script to upgrade a dataset.
+ */
+export const compileUpgradeSequence = new rpc.RequestType<WithProgressId<CompileUpgradeSequenceParams>, CompiledUpgradeSequence, void, void>('compilation/compileUpgradeSequence');
```

Comment:
Our naming convention is usually `XxxParams` and `XxxResult`.  Here it's `CompileUpgradeSequenceParams` and `CompiledUpgradeSequence`. Can you follow the same convention here?

Reply:
Hmmm...and I see below that `EvaluateQueriesParams` and `EvaluationComplete` does not follow the convention either. I still think it's useful to do so in order to know how these two interfaces are related.

I may fix that in a later PR.